src/components/canvas/Box.js

```diff
@@ -4,27 +4,41 @@ import { a, useSpring } from '@react-spring/three'
 import { RoundedBoxGeometry } from 'three/examples/jsm/geometries/RoundedBoxGeometry'
 import { extend } from 'react-three-fiber'
 import useStore from '@/helpers/store'
+import { A11y, useA11y, useUserPreferences } from "@react-three/a11y"
 
 const M = a(MeshDistortMaterial)
 extend({ RoundedBoxGeometry })
 
-const BoxComponent = () => {
-  const router = useStore((s) => s.router)
+const RoundedDarkBox = () => {
+  const { a11yPrefersState } = useUserPreferences()
+  const a11y = useA11y()
   const { color } = useSpring({
-    color: router.route !== '/box' ? 'black' : '#272727',
+    color: a11y.focus || a11y.hover ? '#494949' : '#272727',
   })
+  return (
+    <mesh
+    rotation={[45, 45, 45]}
+  >
+    <roundedBoxGeometry args={[1.5, 1.5, 1.5, 10, 0.1]} />
+  <M distort={a11yPrefersState.prefersReducedMotion ? 0 : 0.4} color={color} />
```

Comment:
I replaced factor by distort as factor didn't seem to exist in the current version and therefore had no effect 
It will leave default ( like before ) if the user is not browsing with prefer reduced motion. 
Otherwise I disable the animation